generators/vue/files-vue.mjs

```diff
@@ -48,17 +49,13 @@ export const vueFiles = {
     },
     {
       condition: generator => generator.microfrontend,
-      ...clientTestBlock,
-      templates: [
-        'spec/app/microfrontends/entities-menu.component.ts',
-        'spec/app/microfrontends/entities-menu.vue',
-        'spec/app/microfrontends/entities-router.ts',
-      ],
+      ...clientApplicationBlock,
+      templates: ['microfrontends/entities-menu.component.ts', 'microfrontends/entities-menu.vue', 'microfrontends/entities-router.ts'],
```

Comment:
We should rename the files to `entities-menu-test.*` and `entities-router-test.*`.
Those files are used for test only.